Remove 'try' from tests, try() from Barrier

psahabu opened the pull request:
@noakesmichael: I'll push tonight unless those mentioned below have objections.

@mppf, @vasslitvinov: I changed your tests to not use `try` as a variable name. Let me know if you would like it changed to something else.

@daviditen, I went with `check()` for all the implementations of `Barrier`. Let me know if it looks funny.

daviditen commented:
The changes in the Barrier module look fine to me.

There's one more test that you probably need to update your branch to get:
test/parallel/taskPar/diten/barrierTry.chpl

I'd be fine with renaming this test to barrierCheck in addition to changing the uses of 'try' in it if you want to do that.

vasslitvinov commented:
@psahabu Let's rename "try" to "tryN" in barrierWF.test-1.chpl - not to "t". That way the variable remains greppable.

psahabu commented:
@daviditen: Thanks, I missed that test. I've updated it and renamed to barrierCheck.

@vasslitvinov: Sounds good. @mppf, I've used `tryN` in your test as well.
